QUEST_LV_0100.tsv

```diff
@@ -3850,30 +3850,30 @@ QUEST_LV_0100_20150317_003849	{memo X}Persuasion of the Battle Commander
 QUEST_LV_0100_20150317_003850	The Battle Commander says he will tell his men to do the rest. Go and report to Knight Titas.
 QUEST_LV_0100_20150317_003851	Report to Knight Titas.
 QUEST_LV_0100_20150317_003852	{memo X}Please be prepared to now go to the Klaipeda.
-QUEST_LV_0100_20150317_003853	Find Search Scout in the marked area
+QUEST_LV_0100_20150317_003853	$Look for the Search Scout in the marked area
 QUEST_LV_0100_20150317_003854	You were asked to tell the troops to assemble before heading to Klaipeda. Tell the Search Scout about the assembly.
-QUEST_LV_0100_20150317_003855	Tell the Search Scout about the shift.
+QUEST_LV_0100_20150317_003855	Tell the Search Scout about the orders{nl}to assemble.
 QUEST_LV_0100_20150317_003856	A Large Kepa suddenly appeared. Defeat it.
 QUEST_LV_0100_20150317_003857	Talk to Search Scout
 QUEST_LV_0100_20150317_003858	Tell the Search Scout about the order to assemble and find the location of the next soldier.
 QUEST_LV_0100_20150317_003859	Defeat the Large Kepa
-QUEST_LV_0100_20150317_003860	Talk to Laimonas
+QUEST_LV_0100_20150317_003860	$Talk to Laimonas
 QUEST_LV_0100_20150317_003861	Let's listen to Laimonas' favor like Titas said.
-QUEST_LV_0100_20150317_003862	Offer worship to the Goddess Statue.
-QUEST_LV_0100_20150317_003863	Offer worship to the Goddess Zemyna Statue. The statue is located at the end of the road on the right of Laimonas.
-QUEST_LV_0100_20150317_003864	You offered worship like Laimonas said. Tell Laimonas about it.
+QUEST_LV_0100_20150317_003862	$Offer a prayer to the Goddess Statue.
+QUEST_LV_0100_20150317_003863	$Offer a prayer to the Statue of Goddess Zemyna. The statue is located at the end of the road on the right of Laimonas.
```

Comment:
Then, to worship. I don't know who made the first revisions and named it "offer worship".